docusaurus/docusaurus.config.js

```diff
@@ -53,8 +53,8 @@ const config = {
                 title: '',
                 logo: {
                     alt: 'Simple, secure and extensible data integration',
-                    src: 'img/logo-dark.png',
-                    srcDark: 'img/logo-light.png',
+                    src: 'img/logo-dark_2.png',
```

Comment:
Hello @Upmitt I'm going to be extra verbose here to try and spread some knowledge about git feedback.  


nit: can we just rename the new assets `logo-dark.png` and remove the old assets?


on pull requests `nit` is used as a short for the English phrase `nitpick` which is to say: this is a suggestion that you can follow... or not.  It is a suggestion that you can simply ignore if you want to.